src/OrchardCore.Modules/OrchardCore.Contents/Drivers/OwnerEditorDriver.cs

```diff
@@ -37,7 +37,8 @@ public override async Task<IDisplayResult> EditAsync(CommonPart part, BuildPartE
         {
             var currentUser = _httpContextAccessor.HttpContext?.User;
 
-            if (!await _authorizationService.AuthorizeAsync(currentUser, StandardPermissions.SiteOwner, part))
+            if (!await _authorizationService.AuthorizeAsync(currentUser, StandardPermissions.SiteOwner)
```

Comment:
Try without the SiteOwner it should work. Or/and add it to the Administrator role.